src/index.ts

```diff
@@ -1,7 +1,14 @@
 import express from "express";
+import personRouter from "./routes/personRouter.js";
 
 const app = express();
 
+app.use(express.json());
+
+app.use("/api/person", personRouter);
```

Comment:
You should make a api folder, with an index file and inside the api folder a personrouter file